cookbooks/gplates-3d.prm

```diff
@@ -0,0 +1,101 @@
+# A simple setup for for using the GPlates interface in a 2d shell. See the
```

Comment:
fix comment to say 3d. Can you add that you probably want a) run in release mode, b) need a powerful machine because it has >1 million unknowns?